packages/table/src/Row.tsx

```diff
@@ -100,7 +100,7 @@ function getIndentLevelStyle(indentLevel: number) {
   `;
 }
 
-const idAllocator = IdAllocator.create('text-input');
+const idAllocator = IdAllocator.create('table');
```

Comment:
should it be “table-row” just to be safe from collisions with other table components?

Reply:
makes sense, updated!